### PR TITLE
Add Leaderboard API

### PR DESCRIPTION
**Pull request by yahyaelganyni1**
in this branch, I implement  the leaderboard API  to the basic HTML&CSS to add username and score to the API

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -1 +1,34 @@
...
+        <h2>Add your score</h2>
+        <form id="form">
+          <input type="text" placeholder="your name" id="user-input" />
+          <input
+            type="number"
+            placeholder="your score"
+            id="score-input"
+            min="1"
+          />
+          <button>submit</button>
```
Comment (vichuge) [inline dist/index.html:29]: - [x] When I submit the new score, the score upload to the API, but this doesn't auto-refresh the score list. I need to refresh the page or something like that to see the new Score, I kindly invite you to add an auto-refresher when you submit a new score. 🐶 
- [OPTIONAL] Why not add `require` on the inputs form? 🌵

File: src/index.js [hunk 2]
```diff
@@ -1 +1,43 @@
...
+    method: 'POST',
+    body: JSON.stringify({
+      user: userInput.value,
+      score: scoreInput.value,
+    }),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  }).then((response) => response.json());
+});
```
Comment (vichuge) [inline src/index.js]: - [x] It looks like you forgot to use `await` in your function, I kindly invite you to add it. 🐱

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your README and PR message are descriptive ✔️

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mimipeshy — APPROVED** [review]
## Hi @yahyaelganyni1 👋🏾 👋🏾  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](<https://microverse.zendesk.com/hc/en-us/articles/1500004088561>)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](<https://airtable.com/shrQAqnBwek5a0O0s>)._

**Pull request merged**